pkgs/by-name/tr/traderepublic-portfolio-downloader/package.nix

```
@@ -0,0 +1,34 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+buildGoModule rec {
+  pname = "traderepublic-portfolio-downloader";
+  version = "0.16.0";
+
+  src = fetchFromGitHub {
+    owner = "dhojayev";
+    repo = "traderepublic-portfolio-downloader";
+    tag = "v${version}";
+    sha256 = "sha256-EXMG+53KKCTbuTLiybO9Zwsog8oOkd1Jrp8+eIFDYoE=";
+  };
+
+  vendorHash = "sha256-VzBBY1mNbT6qHnsy1GE+jWXHZcUo3TNefQixVFF+HYA=";
+
+  postInstall = ''
+    mv $out/bin/public $out/bin/traderepublic-portfolio-downloader
+    rm $out/bin/dev
+    rm $out/bin/example-generator
+  '';
+
+  meta = with lib; {
+    description = "Downloads trade republic portfolio data";
+    homepage = "https://github.com/dhojayev/traderepublic-portfolio-downloader";
+    changelog = "https://github.com/dhojayev/traderepublic-portfolio-downloader/releases/tag/v${version}";
+    license = licenses.gpl3;
+    mainProgram = "traderepublic-portfolio-downloader";
+    maintainers = with maintainers; [ seineeloquenz ];
+    platforms = platforms.linux;
+  };
```

```suggestion
  meta = {
    description = "Downloads trade republic portfolio data";
    homepage = "https://github.com/dhojayev/traderepublic-portfolio-downloader";
    changelog = "https://github.com/dhojayev/traderepublic-portfolio-downloader/releases/tag/v${version}";
    license = lib.licenses.gpl3;
    mainProgram = "traderepublic-portfolio-downloader";
    maintainers = with lib.maintainers; [ seineeloquenz ];
    platforms = lib.platforms.linux;
  };
```
possibly removing `with lib;` here might be an improvement suggestion, see https://github.com/NixOS/nixpkgs/issues/371862

---

Agreed.